data/Shreyansh-09.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Shreyansh Faye",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Newbie in Web Dev learning frontend with neogcamp",
+  "avatar": "https://avatars.githubusercontent.com/u/68677605?v=4",
```

It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/Shreyansh-09.png",
```

---

yeah sure!